test/components/interpolatedColorLegendTests.ts

```diff
@@ -187,4 +188,102 @@ describe("InterpolatedColorLegend", () => {
+
+    legend.orientation("left");
+    assert.isFalse(legend.fixedHeight(), "height is not fixed oriented vertically");
+
+    legend.orientation("right");
+    assert.isFalse(legend.fixedHeight(), "height is not fixed oriented vertically");
+
+    legend.expand(false);
+    assert.isTrue(legend.fixedHeight(), "height is fixed when expand is set to false");
+
+    svg.remove();
+  });
+
+  it("fixed width if expand is set to false or orientation is vertically", () => {
+    let legend = new Plottable.Components.InterpolatedColorLegend(colorScale);
+    legend.renderTo(svg);
+
+    assert.isTrue(legend.fixedWidth(), "width is fixed on default");
+
+    legend.expand(true);
+    assert.isFalse(legend.fixedWidth(), "width is not fixed oriented horizontally");
+
+    legend.orientation("left");
+    assert.isTrue(legend.fixedWidth(), "width is fixed oriented vertically");
+
+    legend.orientation("right");
+    assert.isTrue(legend.fixedWidth(), "width is fixed oriented vertically");
+
+    legend.expand(false);
+    legend.orientation("horizontal");
+    assert.isTrue(legend.fixedWidth(), "width is fixed when expand is set to false");
+
+    svg.remove();
+  });
+
+  it("spams the entire height if oriented vertically and expand is set to true", () => {
+    let legend = new Plottable.Components.InterpolatedColorLegend(colorScale);
+    legend.orientation("left");
+    legend.expand(true);
+    legend.renderTo(svg);
+    assert.closeTo(legend.height(), SVG_HEIGNT, 0.01, "legend height is the same as svg height");
+    svg.remove();
+  });
+
+  it("spams the entire width if oriented horizontally and expand is set to true", () => {
+    let legend = new Plottable.Components.InterpolatedColorLegend(colorScale);
+    legend.expand(true);
+    legend.renderTo(svg);
+    assert.closeTo(legend.width(), SVG_WIDTH, 0.01, "legend width is the same as svg width");
+    svg.remove();
+  });
+
+  it("has default number of swatches by default", () => {
+    let legend = new Plottable.Components.InterpolatedColorLegend(colorScale);
+    legend.renderTo(svg);
+
+    let numSwatches = (<any> legend)._numSwatches;
+    let horzontalSwatches = legend.content().selectAll(".swatch");
+    assert.strictEqual(horzontalSwatches.size(), numSwatches, "there are 10 swatches by default");
+
+    legend.orientation("left");
+    let leftSwatches = legend.content().selectAll(".swatch");
+    assert.strictEqual(leftSwatches.size(), numSwatches, "there are 10 swatches by default");
+
+    legend.orientation("right");
+    let rightSwatches = legend.content().selectAll(".swatch");
+    assert.strictEqual(rightSwatches.size(), numSwatches, "there are 10 swatches by default");
+
+    svg.remove();
+  });
+
+  it("has more swatches than default when expand is true", () => {
+    let legend = new Plottable.Components.InterpolatedColorLegend(colorScale);
+    legend.expand(true).renderTo(svg);
+
+    let numSwatches = (<any> legend)._numSwatches;
```

Comment:
Ditto here. A more interesting test might be one that compares the number of swatches for a given orientation when the value of `expands()` is changed:
* render `InterpolatedColorLegend`
* count number of swatches
* `expands(true)`
* assert number of swatches has increased